### ISSUE TITLE
iinit wrong password for rods user start spawning irodsServer processes 

### ISSUE THREAD

**Issue by kwaegema** [irods/irods#3664]
I logged in as irods user and i ran iinit with a wrong password. A few minutes later, irods went supposedly down, no matter which user, each command returning 
```
remote addresses: 10.141.21.200 ERROR: read_client_server_negotiation_message :: received error message -38000
remote addresses: 10.141.21.200 ERROR: _rcConnect: connectToRhost error, server on test2800.banette.os:1247 is probably down status = -38000 SYS_AGENT_INIT_ERR
remote addresses: 10.141.21.200 ERROR: rcConnect failure SYS_AGENT_INIT_ERR () (-38000) _rcConnect: connectToRhost failed
```

Also in irods log:
```
remote addresses: 10.141.21.200 ERROR: _rcConnect: connectToRhost error, server on test2800.banette.os:1247 is probably down status = -38000 SYS_AGENT_INIT_ERR
remote addresses: 10.141.21.200 ERROR: rcConnect failed -38000, _rcConnect: connectToRhost failed

```
What goes wrong is irodsServer starts spawning processes:
```
irods    16475  0.0  3.4 1022036 138496 ?      Ssl  16:38   0:00 /usr/sbin/irodsServer
irods    16476  0.0  0.1 106640  6912 ?        S    16:38   0:01  \_ /usr/sbin/irodsServer
irods    17896  0.3  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17898  0.3  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17900  0.4  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17902  0.4  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17907  0.4  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17912  0.5  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17915  0.4  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17920  0.5  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17922  0.5  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17930  0.6  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17935  0.6  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17937  0.7  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17945  0.6  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17950  0.8  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17955  1.0  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17960  1.1  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17962  1.2  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17967  1.4  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17969  1.6  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17972  2.2  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17974  3.3  0.7 360808 28920 ?        S    17:16   0:00  |   \_ /usr/sbin/irodsServer
irods    17976  5.0  0.7 360808 28920 ?        S    17:17   0:00  |   \_ /usr/sbin/irodsServer
irods    17978 11.0  0.7 360808 28920 ?        S    17:17   0:00  |   \_ /usr/sbin/irodsServer
```
...
When there are exactly 100 processes, our postgresql won't let anymore connecting:
posgress log:
`FATAL:  remaining connection slots are reserved for non-replication superuser connections`

resulting in an unusable irods cluster. So irodsServer should probably stop spawning new processes.

We are running irods-server-4.2.1.

Thanks !

**Comment by trel** [comment]
Reproduced.   Will investigate and fix ASAP.

**Comment by trel** [comment]
With bench testing, this appears to be fixed by the commit for #3605 (already closed and marked for 4.2.2).

If you can confirm, we will close this as a duplicate.

**Comment by trel** [comment]
closing as duplicate
